docs/docs/components/lifecycle-operator/webhook.md

```diff
@@ -0,0 +1,70 @@
+---
+comments: true
+---
+
+# Keptn Webhooks
+
+Keptn uses
+[Admission Webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)
+to mutate resources.
+To enable the webhook (and therefore Keptn Lifecycle Management)
+for a certain namespace, the namespace needs to be annotated:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: podtato-kubectl
+  annotations:
+    keptn.sh/lifecycle-toolkit: "enabled"  # this line tells the webhook to handle the namespace
+```
+
+The mutating webhook only modifies specifically annotated resources in the annotated namespace.
+When the webhook receives a request for a new pod,
+it looks for the workload annotations:
+
+```yaml
+keptn.sh/workload: "some-workload-name"
+```
+
+The mutation consists in changing the scheduler used for the deployment
```

Comment:
This syntax is problematic.  Maybe "The mutation either replaces the scheduler... with the Keptn Scheduler or adds..."